Dockerfile

```diff
@@ -108,6 +108,8 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/inst
     && pip3 install -U wheel \
     && /root/.local/bin/poetry config virtualenvs.create false \
```

Comment:
should we just convert this over to building a requirements.txt and installing w/ pip as well, instead of using `poetry install`?

Reply:
Probably